### PR TITLE
Expose SQLite VFS option in connection string

### PR DESCRIPTION
**Pull request by arontsang**
Simple PR to expose the VFS option in the connection string.

### CODE REVIEW

**Comment by cincuranet** [comment]
@arontsang This needs to be updated to target `main` branch.

**Comment by arontsang** [comment]
@cincuranet Updated

File: src/Microsoft.Data.Sqlite.Core/SqliteConnectionStringBuilder.cs [hunk 1]
```diff
@@ -269,6 +285,9 @@ public override object? this[string keyword]
                     case Keywords.Pooling:
                         Pooling = Convert.ToBoolean(value, CultureInfo.InvariantCulture);
                         return;
+                    case Keywords.Vfs:
+                        Vfs = Convert.ToString(value, CultureInfo.InvariantCulture);;
```
Comment (cincuranet) [inline src/Microsoft.Data.Sqlite.Core/SqliteConnectionStringBuilder.cs:289]: nit: Two semicolons at the end.

File: src/Microsoft.Data.Sqlite.Core/SqliteConnectionStringBuilder.cs [hunk 2]
```diff
@@ -458,6 +477,9 @@ private void Reset(Keywords index)
                 case Keywords.Pooling:
                     _pooling = true;
                     return;
+                case Keywords.Vfs:
+                    _vfs = string.Empty;
```
Comment (cincuranet) [inline src/Microsoft.Data.Sqlite.Core/SqliteConnectionStringBuilder.cs:481]: Same as above. `null`?

File: src/Microsoft.Data.Sqlite.Core/SqliteConnectionStringBuilder.cs [hunk 3]
```diff
@@ -55,10 +57,11 @@ private enum Keywords
         private bool _recursiveTriggers;
         private int _defaultTimeout = 30;
         private bool _pooling = true;
+        private string? _vfs = string.Empty;
```
[the remark this answers is not in the document]
Comment (cincuranet) [reply src/Microsoft.Data.Sqlite.Core/SqliteConnectionStringBuilder.cs:60]:
```suggestion
        private string? _vfs;
```

File: src/Microsoft.Data.Sqlite.Core/SqliteConnectionStringBuilder.cs (hunk 1 above)
Comment (cincuranet) [reply src/Microsoft.Data.Sqlite.Core/SqliteConnectionStringBuilder.cs:289]:
```suggestion
                        Vfs = Convert.ToString(value, CultureInfo.InvariantCulture);
```

File: src/Microsoft.Data.Sqlite.Core/SqliteConnectionStringBuilder.cs (hunk 2 above)
Comment (cincuranet) [reply src/Microsoft.Data.Sqlite.Core/SqliteConnectionStringBuilder.cs:481]:
```suggestion
                    _vfs = null;
```

**Review by cincuranet — COMMENTED** [review]
Looks like there's no response from the author. But I still think this is a good feature and PR, so I'll take over and finish it.

**Review by cincuranet — APPROVED** [review]

**Pull request merged**